tla-types/src/test/scala/at/forsyte/apalache/tla/typecheck/etc/TestTypeCheckerTool.scala

```diff
@@ -94,4 +94,33 @@ class TestTypeCheckerTool extends FunSuite with BeforeAndAfterEach with EasyMock
       }
     }
   }
+
+  test("the tools runs consumes its output") {
```

Comment:
Is this mean to be something like?

```suggestion
  test("the tools consumes its output") {
```

I think there's one too many `s`s for me to parse this out.

Reply:
Yes, you are right :))